app/models/case/sar/offender.rb

```diff
@@ -292,6 +326,10 @@ def offender_sar?
     true
   end
 
+  def rejected_offender_sar?
```

Comment:
You can call `.rejected?` on a case to check if a case is in the rejected state so this method isn't needed.